kahuna/public/js/components/gr-panel/gr-panel.html

```diff
@@ -35,13 +35,15 @@
                 <div class="image-info__usage-rights drop-menu__items drop-menu__items--right"
                      ng:show="showUsageRights">
 
-                    <gr:usage-rights-editor
+                    <gr-usage-rights-editor
                         gr:usage-rights="ctrl.usageRights"
                         gr:on-save="showUsageRights = false"
                         gr:on-cancel="showUsageRights = false">
-                    </gr:usage-rights-editor>
+                    </gr-usage-rights-editor>
                 </div>
 
+                <gr-downloader gr:images="ctrl.images"></gr-downloader>
```

Comment:
Should this be at the bottom of the panel instead (by the archiver), while we wait for a bar to put all those actions on?